python/tests/conftest.py

```diff
@@ -363,6 +363,43 @@ def make_cfg(name):
             lib.version_store.clear()
 
 
+@pytest.fixture(scope="function")
+def real_s3_credentials():
+    endpoint = os.getenv("ARCTICDB_REAL_S3_ENDPOINT")
+    bucket = os.getenv("ARCTICDB_REAL_S3_BUCKET")
+    region = os.getenv("ARCTICDB_REAL_S3_REGION")
+    access_key = os.getenv("ARCTICDB_REAL_S3_ACCESS_KEY")
+    secret_key = os.getenv("ARCTICDB_REAL_S3_SECRET_KEY")
+    clear = True if str(os.getenv("ARCTICDB_REAL_S3_CLEAR")).lower() in ["true", "1"] else False
```

Comment:
Same comment as for the seed job. Let's make this function reusable.

Reply:
I wouldn't bother. I have an upcoming change that changes all the fixture setup.